account_invoice_alternate_payer/models/account_move.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+
+    _inherit = "account.move"
+
+    alternate_payer_id = fields.Many2one(
+        "res.partner",
+        string="Alternate Payer",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        help="If set, this partner will be the that we expect to pay or to "
+        "be paid by. If not set, the payor is by default the "
```

Comment:
```suggestion
        "be paid by. If not set, the payer is by default the "
```

Reply:
Oh, it is just the American or English. don't worry about it.
https://wikidiff.com/payor/payer